dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -96,7 +96,7 @@ tokenEndpointAuthMethod=Authentication method
 tokenEndpointAuthMethod.desc=Specifies required authentication method.
 
 redirectToRPHostAndPort=Callback host and port number
-redirectToRPHostAndPort.desc=Specifies a callback host and port number.
+redirectToRPHostAndPort.desc=Specifies a callback protocol, host, and port number. For example, http://myhost:8020
```

Comment:
I'd put a period at the end so we're consistent with the other descriptions. I'd also use `https` as the protocol; it's subtle, but it would be a little bit of encouragement/reminder to use HTTPS instead of HTTP.

Reply:
Reviewing on the ID side. Agree with @ayoho on the above suggested edits.